components/Pages/Dashboard/BondingActions/hooks/bondTokens.ts

```diff
@@ -10,7 +10,7 @@ import { createExecuteMessage } from 'util/messages/createExecuteMessage'
 
 export const bondTokens: any = async (
   signingClient: SigningCosmWasmClient,
-  injectiveSigningClient: InjectiveSigningStargateClient,
+  injectiveSigningClient: InjectiveSigningStargateClient | null,
```

Comment:
injectiveSigningClient?: InjectiveSigningStargateClient,
 
 **note for linting: first non nullable items then nullable